src/v2/stack_aggregator.py

```diff
@@ -9,19 +9,16 @@
 import time
 import logging
 
-from abc import ABC, abstractmethod
-from typing import Dict, List, Tuple, Union, Set
+from abc import ABC
```

Comment:
It is not necessary to use ABC(Abstract Base Class). Aggregator is a concrete class now.